docs/release_notes/pending_release.rst

```diff
@@ -5,9 +5,9 @@ XAITK-Saliency Pending Release Notes
 Updates / New Features
 ----------------------
 
-Interfaces
+Implementations
```

Comment:
You seem to have *replaced* the contents of this file. Your notes should be additions, instead.

Reply:
Sorry I was not sure how to fill that up. I reverted my commit and made it an addition. Please let me know what you think.